uml-editor/src/models/jointjs/uml-actor.model.ts

```diff
@@ -1,38 +1,49 @@
-import { dia, util } from '@joint/core'
+import { dia, mvc, shapes, util } from '@joint/core'
 import { CustomJointJSElementAttributes } from './custom-jointjs-element.model'
+import { TextBlock } from './text-block.model'
 
 const legsY = 0.7
 const bodyY = 0.3
 const headY = 0.15
-
-const COLORS = ['#3f84e5', '#49306B', '#fe7f2d', '#ad343e', '#899e8b', '#ede9e9', '#b2a29f', '#392F2D']
+const initWidth = 40
+const initHeight = 80
 
 export class UmlActor extends dia.Element {
-  override readonly markup = [
-    {
-      tagName: 'rect',
-      selector: 'background',
-    },
-    {
-      tagName: 'path',
-      selector: 'body',
-    },
-    {
-      tagName: 'circle',
-      selector: 'head',
-    },
-    {
-      tagName: 'text',
-      selector: 'label',
-    },
-  ]
+  private textBlockMarkup: dia.MarkupJSON | undefined // Placeholder for TextBlock markup
+
+  override initialize(
+    attributes?: shapes.standard.RectangleAttributes,
+    options?: mvc.CombinedModelConstructorOptions<never, this>
+  ) {
+    super.initialize(attributes, options)
+    // Initialize textBlockMarkup with TextBlock markup
+    const tb = new TextBlock()
```

Comment:
Do not use markup over a created class. Instead export the markup in the same file as the class who uses it originally lives